content/influxdb/v2.0/reference/cli/influxd/inspect/export-lp.md

```diff
@@ -33,6 +33,11 @@ influxd inspect export-lp [flags]
 
```

Comment:
Ah, should the description of `--output-path` be updated above?

Reply:
Yeah, you're right. I updated the flag description and added stdout alternatives to all examples.